weblate/machinery/tests.py

```diff
@@ -1023,6 +1042,62 @@ def test_cache(self):
         self.assertEqual(len(responses.calls), 0)
 
 
+@override_settings(MT_LIBRETRANSLATE_API_URL="URL")
```

Comment:
Should be a real URL for tests and should match the one used in `responses.add`:

```suggestion
@override_settings(MT_LIBRETRANSLATE_API_URL="https://libretranslate.com/")
```